db/migrate/20180413193159_require_min_title_events.rb

```diff
@@ -0,0 +1,5 @@
+class RequireMinTitleEvents < ActiveRecord::Migration
+  def change
+    change_column_null :events, :min_title, false, "Recruit"
```

Comment:
I would prefer not to put the constraint in the DB, at least not until the titles are in the database and not hardcoded in the source. 
Also, wouldn't this prevent editing earlier events that may be in production and not have a min_title ? We may need to define one for those that don't currently have one.

Reply:
This should also be 'Unknown', there has to be a constraint as it is a required entity and will crash the current application if there are values in the DB set to nil.